### PR TITLE
Adding logs to Device Component

### PR DESCRIPTION
**Pull request by tPeeush**
Adding logs to Device Component.

Signed-off-by: tPeeush <peeush.thakur@intel.com>

### CODE REVIEW

File: component-samples/device/src/main/java/org/fidoalliance/fdo/sample/DeviceApp.java [hunk 1]
```diff
@@ -104,6 +104,7 @@ public void generateHello() throws IOException {
       final DeviceCredential devCredential = Config.getWorker(DeviceCredentialSupplier.class).get();
       if (devCredential == null) {
         generateDiHello();
+        logger.info("Generating Device Credential file ");
```
Comment (DukeDavis12) [inline component-samples/device/src/main/java/org/fidoalliance/fdo/sample/DeviceApp.java:107]: Please remove whitespace.

File: component-samples/device/src/main/java/org/fidoalliance/fdo/sample/DeviceApp.java [hunk 2]
```diff
@@ -241,6 +244,7 @@ private byte[] generateCsr(PublicKeyType keyType, KeySizeType keySize)
       return pkcs10.getEncoded();
 
     } catch (OperatorCreationException | CertificateEncodingException e) {
+      logger.error("Operation Creation/Certificate Encoding Execution Failed : " + e.getMessage());
```
Comment (DukeDavis12) [inline component-samples/device/src/main/java/org/fidoalliance/fdo/sample/DeviceApp.java:247]: Certificate Creation ?

File: component-samples/device/src/main/java/org/fidoalliance/fdo/sample/DeviceValidityDays.java [hunk 3]
```diff
@@ -4,12 +4,18 @@
...
+
+import org.fidoalliance.fdo.protocol.LoggerService;
 import org.fidoalliance.fdo.protocol.dispatch.ValidityDaysSupplier;
 
 public class DeviceValidityDays implements ValidityDaysSupplier {
+  LoggerService logger =  new LoggerService(DeviceValidityDays.class);
 
   @Override
   public Integer get() throws IOException {
+    logger.info("Validity days for device cert request");
```
Comment (DukeDavis12) [inline component-samples/device/src/main/java/org/fidoalliance/fdo/sample/DeviceValidityDays.java:16]: Change it to Logger.debug() ; Add validity period also in logs.

File: component-samples/device/src/main/java/org/fidoalliance/fdo/sample/DeviceValidityDays.java [hunk 4]
```diff
@@ -4,12 +4,18 @@
...
     return 360 * 10;//the validity days for the device cert request
+
+
```
Comment (DukeDavis12) [inline component-samples/device/src/main/java/org/fidoalliance/fdo/sample/DeviceValidityDays.java:19]: Remove whitespace.

File: component-samples/device/src/main/java/org/fidoalliance/fdo/sample/FdoSysDeviceModule.java [hunk 5]
```diff
@@ -271,6 +274,7 @@ private void exec_cb(String[] args) throws IOException {
       //set the first status check
       createStatus(false, 0, statusTimeout);
     } catch (IOException e) {
+      logger.error("IO Operation Failed");
```
Comment (DukeDavis12) [inline component-samples/device/src/main/java/org/fidoalliance/fdo/sample/FdoSysDeviceModule.java:277]: Exec_CB IO Operation failed + e.getMessage()

File: component-samples/device/src/main/java/org/fidoalliance/fdo/sample/StandardDeviceModule.java [hunk 6]
```diff
@@ -151,6 +152,7 @@ public void send(ServiceInfoModuleState state, ServiceInfoSendFunction sendFunct
     }
 
     if (queue.size() == 0) {
+      logger.warn("Size of queue = 0 ");
```
Comment (DukeDavis12) [inline component-samples/device/src/main/java/org/fidoalliance/fdo/sample/StandardDeviceModule.java:155]: logger.info("Completed DeviceServiceInfo messages");

File: component-samples/device/src/main/java/org/fidoalliance/fdo/sample/StandardMaxServiceInfoSupplier.java [hunk 7]
```diff
@@ -2,14 +2,18 @@
...
 
 
 public class StandardMaxServiceInfoSupplier implements MaxServiceInfoSupplier {
+  private static final LoggerService logger =
+          new LoggerService(StandardMaxServiceInfoSupplier.class);
 
   @Override
   public Integer get() throws IOException {
     DeviceConfig config = Config.getConfig(RootConfig.class).getRoot();
+    logger.info("SviMtu " + config.getSviMtu());
```
Comment (DukeDavis12) [inline component-samples/device/src/main/java/org/fidoalliance/fdo/sample/StandardMaxServiceInfoSupplier.java:16]: SVI MTU value:

File: component-samples/device/src/main/java/org/fidoalliance/fdo/sample/UnsupportedCredReuseFunction.java [hunk 8]
```diff
@@ -5,13 +5,16 @@
...
+import org.fidoalliance.fdo.protocol.LoggerService;
 import org.fidoalliance.fdo.protocol.dispatch.CredReuseFunction;
 
 public class UnsupportedCredReuseFunction implements CredReuseFunction {
+  private static final LoggerService logger = new LoggerService(UnsupportedCredReuseFunction.class);
 
   @Override
   public Boolean apply(Boolean reuseFlag) throws IOException {
     if (reuseFlag) {
+      logger.warn("Don't reuse credentials ");
```
Comment (DukeDavis12) [inline component-samples/device/src/main/java/org/fidoalliance/fdo/sample/UnsupportedCredReuseFunction.java:17]: Device doesn't support credential resuse

File: protocol/src/main/java/org/fidoalliance/fdo/protocol/api/AioRvInfo.java [hunk 9]
```diff
@@ -65,6 +65,7 @@ public void doPost() throws Exception {
 
       if (rviData == null) {
         // if data doesn't exist in DB, create new row and insert into RV_DATA table.
+        logger.info("Insert data into RV_DATA");
```
Comment (DukeDavis12) [inline protocol/src/main/java/org/fidoalliance/fdo/protocol/api/AioRvInfo.java:68]: Inserting

File: protocol/src/main/java/org/fidoalliance/fdo/protocol/api/AllowList.java [hunk 10]
```diff
@@ -49,6 +49,7 @@ public void doPost() throws Exception {
       allowList.setAllowed(true);
       getSession().save(allowList);
     }
+    logger.debug("committing transaction");
```
Comment (DukeDavis12) [inline protocol/src/main/java/org/fidoalliance/fdo/protocol/api/AllowList.java:52]: Committing to AllowList Table.

File: protocol/src/main/java/org/fidoalliance/fdo/protocol/api/AllowList.java [hunk 11]
```diff
@@ -69,6 +70,7 @@ public void doDelete() throws Exception {
 
     if (allowList != null) {
       // delete the row, if data exists.
+      logger.warn("Deleting transaction");
```
Comment (DukeDavis12) [inline protocol/src/main/java/org/fidoalliance/fdo/protocol/api/AllowList.java:73]: Deleted from AllowList table.

File: protocol/src/main/java/org/fidoalliance/fdo/protocol/api/DeviceStateApi.java [hunk 12]
```diff
@@ -8,6 +8,7 @@
 
 public class DeviceStateApi extends RestApi {
 
+
```
Comment (DukeDavis12) [inline protocol/src/main/java/org/fidoalliance/fdo/protocol/api/DeviceStateApi.java:11]: Remove whitespace.

File: protocol/src/main/java/org/fidoalliance/fdo/protocol/api/MfgVoucher.java [hunk 13]
```diff
@@ -50,9 +55,11 @@ public void doPost() throws Exception {
   public void doGet() throws Exception {
 
     String path = getLastSegment();
+    logger.info("Manufacturing Voucher doGet Path : " + path);
```
Comment (DukeDavis12) [inline protocol/src/main/java/org/fidoalliance/fdo/protocol/api/MfgVoucher.java:58]: Manufacturing Voucher SerialNo: " + path

File: protocol/src/main/java/org/fidoalliance/fdo/protocol/api/OnDie.java [hunk 14]
```diff
@@ -38,6 +40,7 @@ public void doGet() throws Exception {
   public void doPost() throws Exception {
 
     String body = getStringBody();
+    logger.info("OnDie doPost body : " + body);
```
Comment (DukeDavis12) [inline protocol/src/main/java/org/fidoalliance/fdo/protocol/api/OnDie.java:43]: Received OnDie POST request;  Add similar one in doGet() also.

File: protocol/src/main/java/org/fidoalliance/fdo/protocol/api/OwnerVoucher.java [hunk 15]
```diff
@@ -111,6 +112,7 @@ public void doGet() throws Exception {
       String text = VoucherUtils.toString(onboardingVoucher.getData());
       getResponse().getWriter().print(text);
     } else {
+      logger.error("Not Found Exception for path : " + path);
```
Comment (DukeDavis12) [inline protocol/src/main/java/org/fidoalliance/fdo/protocol/api/OwnerVoucher.java:115]: Voucher not found for GUID: " + path

File: protocol/src/main/java/org/fidoalliance/fdo/protocol/api/ResellApi.java [hunk 16]
```diff
@@ -43,6 +46,7 @@ public void doPost() throws Exception {
         }
 
       } else {
+        logger.warn("Not found exception for path " + path);
```
Comment (DukeDavis12) [inline protocol/src/main/java/org/fidoalliance/fdo/protocol/api/ResellApi.java:49]: Voucher not found for GUID: " + path

File: protocol/src/main/java/org/fidoalliance/fdo/protocol/api/RvInfo.java [hunk 17]
```diff
@@ -22,6 +24,7 @@ protected void doGet() throws Exception {
 
     String body = rviData.getData().getSubString(1,
         Long.valueOf(rviData.getData().length()).intValue());
+    logger.info("Rv info doGet body : " + body);
```
Comment (DukeDavis12) [inline protocol/src/main/java/org/fidoalliance/fdo/protocol/api/RvInfo.java:27]: RVINFO

File: protocol/src/main/java/org/fidoalliance/fdo/protocol/api/RvInfo.java [hunk 18]
```diff
@@ -30,6 +33,7 @@ protected void doGet() throws Exception {
   public void doPost() throws Exception {
 
     String body = getStringBody();
+    logger.info("Rv info doPost body : " + body);
```
Comment (DukeDavis12) [inline protocol/src/main/java/org/fidoalliance/fdo/protocol/api/RvInfo.java:36]: RVInfo

File: protocol/src/main/java/org/fidoalliance/fdo/protocol/api/SviPackage.java [hunk 19]
```diff
@@ -28,6 +28,7 @@ public void doGet() throws Exception {
       String result = systemPackage.getData().getSubString(1,
           Long.valueOf(systemPackage.getData().length()).intValue());
       getResponse().getWriter().print(result);
+      logger.info("SVI package result : " + result);
```
Comment (DukeDavis12) [inline protocol/src/main/java/org/fidoalliance/fdo/protocol/api/SviPackage.java:31]: .debug()

File: protocol/src/main/java/org/fidoalliance/fdo/protocol/api/SystemResourceApi.java [hunk 20]
```diff
@@ -59,6 +59,7 @@ public void doPost() throws IOException, SQLException {
     // Collect parameter 'filename' from HttpRequest parameter and
     // uploaded file content from InputStream of request.
     String fileName = getParamByValue("filename");
+    logger.info("filename - " + fileName);
```
Comment (DukeDavis12) [inline protocol/src/main/java/org/fidoalliance/fdo/protocol/api/SystemResourceApi.java:62]: Uploaded File: + fileName +  to SystemResource Table.

File: protocol/src/main/java/org/fidoalliance/fdo/protocol/api/To2Blob.java [hunk 21]
```diff
@@ -20,6 +22,7 @@ protected void doGet() throws Exception {
 
       String body = onboardConfig.getRvBlob().getSubString(1,
           Long.valueOf(onboardConfig.getRvBlob().length()).intValue());
+      logger.info("To2 doGet body : " + body);
```
Comment (DukeDavis12) [inline protocol/src/main/java/org/fidoalliance/fdo/protocol/api/To2Blob.java:25]: TO2  Body:

File: protocol/src/main/java/org/fidoalliance/fdo/protocol/api/To2Blob.java [hunk 22]
```diff
@@ -29,6 +32,7 @@ protected void doGet() throws Exception {
   @Override
   public void doPost() throws Exception {
     String body = getStringBody();
+    logger.info("To2 doPost body : " + body);
```
Comment (DukeDavis12) [inline protocol/src/main/java/org/fidoalliance/fdo/protocol/api/To2Blob.java:35]: TO2 body

File: protocol/src/main/java/org/fidoalliance/fdo/protocol/db/To0Scheduler.java [hunk 23]
```diff
@@ -115,9 +115,8 @@ private void onInterval() {
       To2AddressEntries addressEntries = new To2BlobSupplier().get();
       OnboardingConfig onboardConfig = new OnboardConfigSupplier().get();
       for (OnboardingVoucher onboardingVoucher : list) {
-        if ((onboardingVoucher.getTo0Expiry() == null
-            || now.after(onboardingVoucher.getTo0Expiry()))
-                && onboardingVoucher.getTo2CompletedOn() == null) {
+        if (onboardingVoucher.getTo0Expiry() == null
+            || now.after(onboardingVoucher.getTo0Expiry())) {
```
Comment (DukeDavis12) [inline protocol/src/main/java/org/fidoalliance/fdo/protocol/db/To0Scheduler.java:120]: Revert changes in this file.

File: protocol/src/main/java/org/fidoalliance/fdo/protocol/StandardMessageDispatcher.java [hunk 24]
```diff
@@ -509,20 +509,6 @@ protected void doTo0OwnerSign(DispatchMessage request, DispatchMessage response)
       throw new InvalidMessageException("Voucher rejected due to untrusted key or guid");
     }
 
-    String guid = Mapper.INSTANCE.readValue(to0d.getVoucher().getHeader(),
-            OwnershipVoucherHeader.class).getGuid().toString();
-
-    try {
-      To2RedirectEntry storedRedirectEntry = getWorker(RvBlobQueryFunction.class).apply(guid);
-      boolean verifiedRedirect = cs.verify(storedRedirectEntry.getTo1d(), ownerPublicKey);
```
Comment (DukeDavis12) [inline protocol/src/main/java/org/fidoalliance/fdo/protocol/StandardMessageDispatcher.java:517]: Revert changes in this file.

**Review by DukeDavis12 — CHANGES_REQUESTED** [review]
Please address the comments.

**Review by DukeDavis12 — DISMISSED** [review]

**Review by KiranSukhavasi — APPROVED** [review]

**Pull request closed without merging**